test/integration/index.js

```diff
@@ -35,6 +36,7 @@ const cleanup = async () => {
         },
         { retries: 3 }
       )
+      // await deleteStaleEnvironments()
```

Comment:
Removed?

Reply:
No just wanted @suevalov to show where I could place that script and do not want run it unless it is approved